addons/website_blog/views/website_blog_templates.xml

```diff
@@ -245,7 +245,7 @@
                     <p class="post-meta text-muted text-center" t-if="len(blog_post.tag_ids)">
                         <span class="fa fa-tags"/>
                         <t t-foreach="blog_post.tag_ids" t-as="one_tag">
-                            <a class="label label-primary mr8" t-attf-href="#{blog_url(tag=one_tag.id)}" t-esc="one_tag.name"/>
+                            <a class="label label-primary mr8" t-attf-href="#{blog_url(tag=slug(one_tag))}" t-esc="one_tag.name"/>
```

Comment:
strange to add a slug while the comment say ["# retrocompatibility to accept tag as slug"](https://github.com/odoo/odoo/pull/30832/files#diff-43da97601625bf08baa784f17cd695f4R110)